modules/docs/arrow-docs/docs/docs/datatypes/store/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-layout: docs
```

Comment:
please keep this change. And also, if you could add `/` at the end of the permalink to fix this, day, moore, and sum, it'd be awesome.

`permalink: /docs/datatypes/store/`

Reply:
Done :+1: